Description: FTBFS on kFreeBSD
 Fix preprocessor environment detection to properly recognize kFreeBSD.
Author: Obey Arthur Liu <arthur@milliways.fr>
Last-Update: 2010-02-28
--- a/libsrc/Dk/Dksystem.h
+++ b/libsrc/Dk/Dksystem.h
@@ -141,7 +141,7 @@
 #endif
 
 #include <errno.h>
-#if !defined(linux) && !defined(__APPLE__) && !defined (WIN32) && !defined (__CYGWIN__) && !defined(__FreeBSD__) && !defined (__cplusplus)
+#if !defined(linux) && !defined(__APPLE__) && !defined (WIN32) && !defined (__CYGWIN__) && !defined(__FreeBSD__) && !defined (__cplusplus) && !defined(__GLIBC__)
 extern char *sys_errlist[];
 extern int sys_nerr;
 #endif
